python/python/glide/async_commands/redis_modules/search/optional_params.py

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+
+
+class FieldFlags(Enum):
```

Comment:
FieldFlags sounds like there are multiple or different flags for fields. maybe change it to FieldsFlag, or to FieldsOption